### DISCUSSION TITLE
Column in another table - depict a certain row [Aug 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3734] [Discord Threads]
> 🙋‍♂️ **torwag**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### what do you mean by depict a certain row ?   
####    
#### If tables are related already then other column can be brought in by using 'lookup'  
#  
> 💁 **o1lab**  
#### LookUp column should do it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1011969823141535784)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
